ftw/contacts/profiles/default/types/ftw.contacts.Contact.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0"?>
+<object name="ftw.contacts.Contact" meta_type="Dexterity FTI"
+        i18n:domain="ftw.contacts" xmlns:i18n="http://xml.zope.org/namespaces/i18n">
+
+  <!-- Basic metadata -->
+  <property name="title" i18n:translate="">Contact</property>
+  <property name="description" i18n:translate=""></property>
+  <property name="icon_expr"></property>
+  <property name="allow_discussion">False</property>
+  <property name="global_allow">False</property>
+  <property name="filter_content_types">True</property>
+    <property name="allowed_content_types" />
+
+  <!-- schema interface -->
+  <property name="schema">ftw.contacts.contents.contact.IContact</property>
+
+  <property name="behaviors">
+      <element value="plone.app.dexterity.behaviors.metadata.ICategorization"/>
+      <element value="plone.app.dexterity.behaviors.metadata.IPublication"/>
+      <element value="plone.app.dexterity.behaviors.metadata.IOwnership"/>
```

Comment:
I think that this adds too many fields which will never be used.
Let's strip it down to the important fields; integrator can add missing behaviors for their use case at any time in their policy code.

Lets have these:
```xml
        <element value="plone.app.dexterity.behaviors.metadata.IBasic" />
        <element value="plone.app.content.interfaces.INameFromTitle" />
        <element value="collective.dexteritytextindexer.behavior.IDexterityTextIndexer" />
```